pkg/backends/datadog/datadog.go

```diff
@@ -279,10 +278,13 @@ func NewClient(apiEndpoint, apiKey string, metricsPerBatch uint, clientTimeout,
 		}).DialContext,
 		MaxIdleConns:    50,
 		IdleConnTimeout: 1 * time.Minute,
+		// A non-nil empty map used in TLSNextProto to disable HTTP/2 support in client.
+		// https://golang.org/doc/go1.6#http2
+		TLSNextProto: map[string](func(string, *tls.Conn) http.RoundTripper){},
 	}
-	if err := http2.ConfigureTransport(transport); err != nil {
-		return nil, err
-	}
+	//if err := http2.ConfigureTransport(transport); err != nil {
+	//	return nil, err
+	//}
```

Comment:
Just remove these lines and later we will just revert the commit to restore the functionality.